qiskit_cold_atom/fermions/fermion_simulator_backend.py

```diff
@@ -20,9 +20,9 @@
 
 from qiskit.providers.models import BackendConfiguration
 from qiskit.providers import Options
-from qiskit.providers.aer import AerJob
 from qiskit import QuantumCircuit
 from qiskit.result import Result
+from qiskit_aer import AerJob
```

Comment:
Why the change? I though `qiskit.providers.aer` was the right import location.

Reply:
My pylint was complaining about the old import and when calling `from qiskit import Aer`, a `PendingDeprecationWarning` triggers that recommends to import from the qiskit-aer package directly, so I thought it made sense to do this for all aer-related imports.